### DISCUSSION TITLE
Help Setting Up

### DISCUSSION THREAD

**Discussion by E0697** [jing332/tts-server-android#184] [Q&A]
I'm trying to set the app up but I can't figure it out and would really appreciate the  help.
I need to use two languages, I've added them in a group. But what should I do next? I tried the Azure plugin, but what should the key and region be?

**Comment by fredphoesh** [comment]
You and me both... IMPOSSIBLE to configure this without instrucitons.

**Comment by fredphoesh** [comment]
@E0697 Tetsy app
